aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -181,6 +181,12 @@
      */
     private final String endpoint;
 
+    /**
+     * This is the hostname and port of the proxy, if any, to be used for connecting to AWS IoT
+     */
+    private static String proxyHost;
```

Comment:
Why are these static? Ideally these are data that lives just for a connect operation. Either they should be scoped to the lifecycle of a connection or instance variables.